frontend/src/pages/studentLesson/BarGraph.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import Paper from '@material-ui/core/Paper';
+import { makeStyles } from '@material-ui/core/styles';
+import { Bar } from 'react-chartjs-2';
+import Title from '../_shared/Title';
+
+const useStyles = makeStyles((theme) => ({
+  histogram: {
+    display: 'flex',
+    justifyContent: 'center',
+    component: 'h2',
+    variant: 'h6',
+    textAlign: 'center',
+  },
+}));
+
+function onlyUnique(value, index, self) {
+  return self.indexOf(value) === index;
+}
+
+// TODO (working on this): IF we want to implement all question types
+// const calculateDistribution = (
+//   questionGroupAttempts,
+//   questionGroupIds,
+//   amoutnQuestions
+// ) => {
+//   let correctPerQuestion = Array(amountQuestions).fill(0);
+//   let incorrectPerQuestion = Array(amountQuestions).fill(0);
+
+//   let missedPerQuestion = Array(amountQuestions).fill(0);
+
+//   questionGroupAttempts.forEach((questionGroupAttempt) => {
+//     if (questionGroupAttempt.isCompleted) {
+//       questionGroupAttempt.questionAttempts.forEach((questionAttempt) => {
+//         const indexCorrect = questionIds.findIndex(
+//           (el) => el === questionAttempt.questionId && questionAttempt.correct
+//         );
+//         correctPerQuestion[indexCorrect] += 1;
+//         const indexIncorrect = questionIds.findIndex(
+//           (el) => el === questionAttempt.questionId && questionAttempt.incorrect
+//         );
+//         incorrectPerQuestion[indexIncorrect] += 1;
+//         const indexMissed = questionIds.findIndex(
+//           (el) => el === questionAttempt.questionId && questionAttempt.missed
+//         );
+//         missedPerQuestion[indexMissed] += 1;
+//       });
+//     }
+//   });
+//   return [correctPerQuestion, incorrectPerQuestion, missedPerQuestion];
+// };
+
+// TODO (working on this): IF we want to implement all question types
+// const getLabels = (amountQuestions) => {
+//   const labels = [];
+//   for (let i = 0; i < amountQuestions; i++) {
+//     labels.push('Vraaggroep ' + parseInt(i + 1));
+//   }
+//   return labels;
+// };
+
+const BarGraph = (props) => {
+  const { questionAttempts, title } = props;
+
+  let questionGroupInfo = [];
```

Comment:
kan `const` worden

Reply:
In dit geval niet, hij wordt geredefineerd met een filter verderop